ios/RNPrint/RNPrint.m

```diff
@@ -94,6 +94,12 @@ -(void)launchPrint:(NSData *) data
     if(options[@"isLandscape"]) {
         _isLandscape = [[RCTConvert NSNumber:options[@"isLandscape"]] boolValue];
     }
+
+    if(options[@"jobName"]) {
+        _jobName = [RCTConvert NSString:options[@"jobName"]];
+    } else {
+        _jobName = @"Document";
```

Comment:
Does it make sense to default to the current implemenation? `[_filePath lastPathComponent]`

Reply:
Wanted to keep the behavior on the android and iOS the same if no jobName is passed.

Reply:
Updated README.